### PR TITLE
fix #2330 track helpshift origin with helpshift tags and mixpanel properties

### LINKED ISSUE #2330: Track origin of new Helpshift conversations
A new Helpshift conversation can be started from the login screen after an error, on the help screen or in the settings screen after login. We must track the origin (have a look at Helpshift analytics or simply use a tag)

### PR DESCRIPTION
**Pull request by maxme**
fix #2330 track helpshift origin with helpshift tags and mixpanel properties

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.java [hunk 1]
```diff
@@ -59,8 +63,9 @@ public void onClick(View v) {
                             SignInFragment.ENTERED_URL_KEY));
                     HelpshiftHelper.getInstance().addMetaData(MetadataKey.USER_ENTERED_USERNAME, extras.getString(
                             SignInFragment.ENTERED_USERNAME_KEY));
+                    origin = (Tag) extras.get(HelpshiftHelper.ORIGIN_KEY);
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.java]: Should this (and line 78 below) check `extras.containsKey(HelpshiftHelper.ORIGIN_KEY)` before reading `origin` to ensure `origin` is never set to null?

Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.java]: a null `origin` is considered an "unknown origin" (and it should not happen). https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2330-track-helpshift-origin/WordPress/src/main/java/org/wordpress/android/util/HelpshiftHelper.java#L95-L97a

Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.java]: :+1:

**Comment by nbradbury** [comment]
:shipit:

**Pull request merged**